### ISSUE TITLE
carrousel plugin does not set image height in safari/IE

### ISSUE THREAD

**Issue by josjevv** [jrief/djangocms-cascade#124]
Probably has to do with the picture logic. The picture source elements are properly scaled, the img tag isn't. 

according to this logic: https://github.com/jrief/djangocms-cascade/blob/master/cmsplugin_cascade/bootstrap3/utils.py#L161

it seems to look at the last bp in the for loop, but only uses image height if it's relative, otherwise it keep the aspect ratio of the origin. In carrousel this is defined in px. 

could we also get this behaviour for px defined heights? Or does this break other components?

**Comment by jrief** [comment]
In order to understand this correctly, could you please append a screenshot.

**Comment by josjevv** [comment]
Ok, browsers that support the `picture` element (e.g. firefox) honour the height set in the carrousel (300px), image is cropped:

![carrousel-firefox](https://cloud.githubusercontent.com/assets/1464875/11975874/7b924c66-a972-11e5-8624-a3dbd379aa42.png)

Safari and IE don't so they use the `img` element, where the original aspect ratio is maintained and the height set in carrousel is ignored:
![carrousel-safari](https://cloud.githubusercontent.com/assets/1464875/11975922/0ad8d8fe-a973-11e5-9895-514b1d91576f.png)

**Comment by jrief** [comment]
Can you please retry using a polyfill. In some of my projects I use [Picturefill](http://scottjehl.github.io/picturefill) version 2.2 which works quite well. I haven't tested version 3 yet.

Could you also please look up the versions of Safari and IE and check them against the listed features here http://caniuse.com/

**Comment by josjevv** [comment]
Hi, I tried both latest 2.3.1 and 3.01 without success. It may be that it doesn't support the max-resolution media query but I am not sure.

from caniuse.com:
* IE9-11 supports the dpi unit, but does not support dppx or dpcm units
* Safari has partial support (using -webkit- prefix) and supports the non-standard min/max-device-pixel-ratio

Anyway, I think the fallback image should be generated correctly anyhow. For now, I patched it quick and dirty using some jQuery:

```
      $(document).ready(function() {
        $('.carousel .item > picture').each(function(index, el) {
          $('img', el).attr('src', $('source:first', el).attr('srcset'));
        });
      });
```

If you want, I can write a patch to use the height set in px (e.g. in carousel) is being used for the fallback img tag as well.
